APITests/utils.py

```diff
@@ -376,10 +379,10 @@ def login_synapse(self) -> synapseclient.Synapse:
         Returns:
             synapse object
         """
-        auth_token = self.get_access_token()
+        #auth_token = self.get_access_token()
         try:
-            syn = synapseclient.Synapse()
-            syn.login(authToken=auth_token)
```

Comment:
This isn't necessary, since the python client automatically reads in `SYNAPSE_AUTH_TOKEN`